content/english/python-earsketch/8-activity-4-add-effects-to-your-songs/_index.md

```diff
@@ -2,15 +2,12 @@
 title: "Activity 4 - Add effects to your song"
 description: "Setting up headphones"
 weight: 9
-prereq: "Python Basics: Print, Comments, Functions"
-difficulty: "Intermediate"
 draft: false
 ---
-<iframe width="100%" height="600px" src="https://www.youtube.com/embed/R_eo0B8qDW0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 Effects in EarSketch
 -------------------------------------
-**Effects** allows the producer to alter the sound of the audio clip
+<b>Effects</b> allow the producer to alter the sound of the audio clip
```

Comment:
```suggestion
**Effects** allow the producer to alter the sound of the audio clip
```

Reply:
That change doesn't render as bold with hugo

Reply:
Hmm that's weird. :(